library/blivet.py

```diff
@@ -801,6 +807,8 @@ def _create(self):
 
         use_vdo = self._volume['deduplication'] or self._volume['compression']
         use_lvmraid = self._volume['raid_level']
+        if use_lvmraid == "striped":
+            use_lvmraid = "raid0"
```

Comment:
This actually won't work because blivet doesn't support raid0 with LVM. This is a bug in blivet*, so this actually should be reversed for LVM RAID and we should replace `raid0` with `striped`.

*) To be more precise, it's a missing feature, support for `lvcreate --type raid0` was added later and we never added it to blivet.